### ISSUE TITLE
[WOTLK] The Missing Diplomat - Cannot Complete

### ISSUE THREAD

**Issue by jimmybrancaccio** [cmangos/issues#1753]
**Current behavior**: 

After obtaining The Missing Diplomat quest (https://www.wowhead.com/quest=1265/the-missing-diplomat) and heading over to Sentry Point. You can see and talk to Archmage Tervosh but nothing happens.

![2018-11-16 09_48_47-world of warcraft](https://user-images.githubusercontent.com/405585/48631706-c41d2780-e984-11e8-98d0-c00c6d8cc7f3.png)

**Expected behavior**: 

You should be able to walk into the tower, maybe walk around inside a little which will mark the 'Sentry Point Explored' as completed which will then allow you to talk to Archmage Tervosh. Here's a YouTube video I found which shows what should be happening (start watching at around 5:10):

https://www.youtube.com/watch?v=6XeuwJnqSd4

**Steps to reproduce**:

1. Obtain quest The Missing Diplomat  (https://www.wowhead.com/quest=1265/the-missing-diplomat)
2.  Walk or mount up and get over to Sentry Point
3. Enter the tower at Sentry Point and Archmage Tervosh should be there, if not, wait a few minutes, he should show up.
4. Entering the tower should be enough to trigger the completion of Sentry Point Explored which in turn should make the question mark over Tervosh's head yellow.

**Client version**: 

3.3.5a

**Commit hash**: 

653dea9d892da7170afc33bf37ee6911ce8dc8b5

**Database version**: 

3aee4927d69b33aa5270b3d871d71cc727d02149

**Operating system**: 

Ubuntu 14.04

[//]: # (If this is a crash report, include the crashlog from a debug build with https://gist.github.com/)

**Comment by Grz3s** [comment]
\src\game\AI\ScriptDevAI\scripts\kalimdor\dustwallow_marsh.cpp
```code
@@ -1226,6 +1226,7 @@ bool AreaTrigger_at_sentry_point(Player* pPlayer, const AreaTriggerEntry* /*pAt*
        if (Creature* pTervosh = pPlayer->SummonCreature(NPC_TERVOSH, -3476.51f, -4105.94f, 17.1f, 5.3816f, TEMPSPAWN_TIMED_DESPAWN, 60000))
        {
            pTervosh->CastSpell(pTervosh, SPELL_TELEPORT_VISUAL_2, TRIGGERED_OLD_TRIGGERED);
+            pPlayer->RewardPlayerAndGroupAtEventExplored(QUEST_MISSING_DIPLO_PT14, pTervosh);

            if (Creature* pGuard = GetClosestCreatureWithEntry(pTervosh, NPC_SENTRY_POINT_GUARD, 15.0f))
            {
```
should be added soon

**Comment by MantisLord** [comment]
The core change proposed above would do the job, however this quest currently works in TBC because of this change here I believe:
https://github.com/cmangos/tbc-db/commit/0507d98106b6f748c85e23660a7ab173a6cd49b7#diff-7bb3b090ddfcf8db095c6fb5b7ff99f2R3

**Comment by Grz3s** [comment]
@MantisLord 
well... yes adding your part will fix this quest.
however .. it makes next one ..
 not to start at all...  (involves the same npc)
Can you pls look if next one "the missing diplomat" -- Subdue Private Hendel. works on your side?

**Comment by Grz3s** [comment]
no idea to be honest...
it does the job...
Fix added:
https://github.com/cmangos/wotlk-db/commit/b1345b5696ce3cfad64a256053b33d8ee0e70711

Can be closed...

**Comment by jimmybrancaccio** [comment]
Closing this out as resolved in cmangos/wotlk-db@b1345b5.
